themes/startwords/assets/scss/article/_notes.scss

```diff
@@ -3,128 +3,208 @@
+            background-size: contain;
 
         }
 
-        /* mirror image version for left side of the page */
+
+        /* mirror image version for notes on the left side of the page */
         &.flip {
             -webkit-clip-path: polygon(70px 0, 100% 34%, 100% 100%, 0 100%, 0 11%);
             clip-path: polygon(70px 0, 100% 34%, 100% 100%, 0 100%, 0 11%);
 
             &::before {
-                left: rem(25px);
+                left: $close-icon-margin;
             }
 
             a[role=doc-backlink] {
-                left: rem(57px);
+                left: $close-icon-margin;
             }
         }
+    }
 
+     /* styles for footnotes as they appear in endnotes */
+    &.endnotes {
 
-    }
+        ol {
+            margin: 0;
 
+            li {
+                position: relative;
+                background: $off-white;
+                color: $dark-grey;
+                width: auto;
+                height: auto;
+                bottom: auto;
+                right: auto;
+                opacity: 1;
+                padding: 0;
+                -webkit-clip-path: none;
+                clip-path: none;
+                pointer-events: auto;
+
+                &::before {
+                    /* position counter for endnote display */
+                    display: inline-block;
+                    float: left;
+                    padding-right: 2px;
+                    vertical-align: top;
+                    font-size: 70%;
+                    font-weight: 600;
+                    font-family: $font-serif;
+                    position: relative;
+                    width: auto;
+                    right: auto;
+                    left: auto;
+                    top: auto;
+                }
+
+                a[role=doc-backlink] {
+                    position: relative;
+                    width: auto;
+                    color: $red;
+                    background: none;
+                    display: inline;
+                    top: auto;
+                    right: auto;
+                    left: auto;
+                }
+
+                p {
+                    padding-top: 0;
+                }
+
+                a {
+                    color: $dark-purple;
+                }
+            }
+        }
+    }
 }
 
 a.footnote-ref {
     color: $red;
     text-decoration: none;
     font-weight: 600;
     font-family: $font-serif;
-
-    /* extend over preceding text */
-    padding-left: 4rem;
-    margin-left: -4rem;
-    padding-bottom: 8px;  /* NOTE: not sure how to make relative to text */;
-
-    &:hover {
-        border-bottom: 2px solid transparent;
-        border-image: linear-gradient(to right, white, $red);
-        border-image-slice: 1;
+    position: relative;
+    /* extend left to make a larger touch target */
+    padding-left: rem(100px);
+    margin-left: rem(-100px);
+    padding-bottom: 0.75rem;
```

Comment:
it was confusing to me that the touch target doesn't correspond to the entire underline; there's an invisible point about halfway along its length at which the link stops being interactive.

Reply:
Ha, whoops. I gave the underline the padding and _also_ the correct width, so it was double the size it was supposed to be. Thanks for catching it.